server/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -19,6 +19,43 @@
 
 package org.elasticsearch.indices;
 
+import static java.util.Collections.emptyList;
```

Comment:
did something change with formatting the imports in your IDE? I've always seen static imports as going last

Reply:
Fascinating. I didn't intentionally change anything, I'll look into that and see if I can undo whatever happened there.